src/drivers/general/queries/IrqlNotSaved/IrqlNotSaved.ql

```diff
@@ -10,11 +11,12 @@
  * @impact Insecure Coding Practice
  * @repro.text This function has a parameter annotated \_IRQL\_saves\_, but does not have the system IRQL saved to it.
  * @owner.email sdat@microsoft.com
- * @kind problem
+ * @opaqueid C28158
  * @problem.severity warning
  * @precision medium
  * @tags correctness
  *       wddst
```

Comment:
Extra tab?

Reply:
At "wddst"? In that case, it's intentional spacing to match SDL style.